airbyte-integrations/connectors/source-us-census/source_us_census/spec.json

```diff
@@ -0,0 +1,36 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/uscensus",
```

Comment:
Nit: The URL should be `https://docs.airbyte.io/integrations/sources/us-census`, just to be consistent with others and slightly more readable.

Our code generator will take care of this part in the future.